### ISSUE TITLE
[Bug] Notifications are constantly fired

### ISSUE THREAD

**Issue by IamKayrox** [YTVanced/Vanced#1124]
**Bug description**
The app starts to constantly fire the same notifications over and over again. These correspond to subscription notifications.

**Variant**
Non-Root

**Vanced version**
Replace me with the exact version of vanced you are using. It is mandatory to report issues from the latest available versions only. For example, v16.20.35

**Android version**
Android 9 

**Device**
HTC U11

**Steps to Reproduce**
I don't know what it causes it but when it starts to do this it never stops until I clear the app data.

**Expected behavior**
Notifications being fired only once.

_Attach images or videos or logcats in case of crashes if possible. These can be helpful in solving the issue._

**Comment by milindgoel15** [comment]
Duplicate of #1107
